source/ota.c

```diff
@@ -1854,6 +1856,9 @@ static DocParseErr_t initDocModel( JsonDocModel_t * pDocModel,
 static OtaErr_t validateUpdateVersion( const OtaFileContext_t * pFileContext )
 {
     OtaErr_t err = OtaErrNone;
+    AppVersion32_t newVersion;
```

Comment:
Is there a need to create this variable if it's only used for debug printing? In the print statement below, can you use 

"pFileContext->updaterVersion.u.x.minor" instead of "appFirmwareVersion.u.x.minor"?

This would be better than having to void the variable when debug printing is disabled.

Reply:
Since `updaterVersion` is only `uint32_t` it has to be first converted to the union `AppVersion32_t` by doing `newVersion.u.unsignedVersion32 = pFileContext->updaterVersion`
without this the formatting doesn't work.